pkg/apis/work/v1alpha2/zz_generated.deepcopy.go

```diff
@@ -31,6 +31,27 @@ func (in *AggregatedStatusItem) DeepCopy() *AggregatedStatusItem {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
This is already updated in fc20a6c040239a30f9f5467e87a85bf91bda0486

Reply:
@iawia002 thanks, it is fixed.